README.md

```diff
@@ -4,7 +4,17 @@
 
 [![Circle CI Status](https://img.shields.io/circleci/project/apiaryio/polls-app.svg)](https://circleci.com/gh/apiaryio/polls-app/tree/master)
 
-This is a Swift iOS client for the [Polls API](https://github.com/apiaryio/polls-api). A simple application allowing users to view polls and vote in them.
+This is a Swift iPhone and iPad client for the [Polls API](https://github.com/apiaryio/polls-api). A simple application allowing users to view polls and vote in them.
```

Comment:
The last sentence is actually a fragment. I think you can combine the two with a dash of some sort.